source/server/admin/admin.cc

```diff
@@ -615,24 +631,56 @@ ProtobufTypes::MessagePtr AdminImpl::dumpEndpointConfigs() const {
   return endpoint_config_dump;
 }
 
+ProtobufTypes::MessagePtr
+AdminImpl::dumpUnreadyTargetsConfigs(const absl::optional<std::string>& mask) const {
+  if (mask.has_value()) {
+    const std::string& mask_string = mask.value();
+    if (mask_string == "listener") {
+      return dumpListenerUnreadyTargetsConfigs();
+    }
+    // More options for unready targets config dump.
+  }
```

Comment:
If no mask, dump everything?

Reply:
I agree that this method is designed to help dump these information, but I am not sure what "everything" should contain here? Listener, cluster and something else, since we need to collect information from each of them.